### ISSUE TITLE
More performance and accessibility 

### ISSUE THREAD

**Issue by sajasbana** [nopSolutions/nopCommerce#5057]
Dears,
I check to demo for the store
Is is possible to check some performance and accessibility issues to make the website more better?
I used google light house to check in desktop and mobile 
![image](https://user-images.githubusercontent.com/34976854/94358553-a190e500-00aa-11eb-9dbf-c7ca641f4b79.png)
![image](https://user-images.githubusercontent.com/34976854/94358652-3bf12880-00ab-11eb-8ed4-64dd6429473c.png)
![image](https://user-images.githubusercontent.com/34976854/94358656-49a6ae00-00ab-11eb-886c-98a5abae0660.png)

for example for images
- is it possible to use picture tag with data-set, and in admin to add feature to add images in different sizes
- lazyloading
for js files
- use async, defer keywords if possible

please try to see result in light house and if there any thing you can do regard this report

![image](https://user-images.githubusercontent.com/34976854/94358612-03e9e580-00ab-11eb-899c-2e9f910cddde.png)

Thanks for you time

**Comment by dylanhthomas** [comment]
Thanks for noticing these =)

Several accessibility issues have been assigned, and a few others have outstanding PRs. Why not take a look and upvote those if you think they would help.

#5022 - lack of visual focus indicator
#5023 - no skip link/bypass block
#5024 - insufficient contrast
#5051 - Show dropdown menus on focus
#5052 - Make mobile menu toggle a <button> element
#5053 - Add aria-live attribute to notifications

**Comment by RomanovM** [comment]
Thanks, we will take into account the info from this report and create specific work items if necessary.
